tooling/cli/lib/parse-translations.js

```diff
@@ -34,13 +34,10 @@ const KEYS_WITH_PLURALS = [];
 
   gatherPluralizedKeys();
 
-  const i18nStore = parser.get();
-  // en is default lang and lms is default namespace, so all keys used should be in that object
-  const i18nNSObject = i18nStore.en && i18nStore.en.lms ? i18nStore.en.lms : {};
-  const usedTranslationKeys = Object.keys(i18nNSObject);
+  const usedTranslationKeys = Object.keys(sourceDefaultNamespace);
```

Comment:
@AlexONeillTI if we're assigning `sourceDefaultNamespace` to `usedTranslationKeys`, it'd seem like we're including the entire translation file rather than the translation keys used in the actual project. That seem accurate?

Reply:
Yes it does actually, but i18nObject is only picking up the following:
`i18nStore:  {
  en: {
    lms: {
      null: '',
      '(empty string)': '',
      finished: '',
      cancelled: '',
      div: '',
      img: '',
      li: '',
      a: ''
    }
  }
}`

Reply:
@AlexONeillTI Believe the `dist` path is correct -- can run `npm run build` to verify that's where the project is compiled. Based on the output of `i18nObject` though, looks like it's possible there's a key or something with invalid characters, causing html tags to get picked up as keys instead. Would log `fileContents` there and see what that looks like.